client/pages/home.jsx

```diff
@@ -1,10 +1,97 @@
-import React from 'react';
-import HelloWorld from '../components/hello-world';
+import React, { useState } from 'react';
+// import NavBar from '../components/nav-bar';
+import Banner from '../components/banner';
+import { Carousel } from '../components/carousel';
+import kalopsiaLogo from '../../images/kalopsia-logo.png';
+import kalopsia from '../../images/kalopsia.png';
 
 export default function Home(props) {
+  const [click, setClick] = useState(false);
```

Comment:
`click` isn't very descriptive. What are you trying to track with this variable? It looks like it's "show/hide"? So `isOpen` or `isVisible`?